src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/ModuleToken.cs

```diff
@@ -64,16 +65,12 @@ public bool Equals(ModuleToken moduleToken)
             return Module == moduleToken.Module && Token == moduleToken.Token;
         }
 
-        public int CompareTo(ModuleToken other)
+        public int CompareTo(ModuleToken other, TypeSystemComparer comparer)
```

Comment:
`TypeSystemComparer comparer` parameter is not used

Reply:
Removed.